Gh 2939 sc gas price (#3077)
aeternity/aeternity: docs/release-notes/next/GH-2939_sc_gas_price.md

* State Channel FSM: fixes fee computation according to transaction size.
  Until now close mutual transactions were expecting a fee to be provided by

>   the initiating party or a default value of 20 000 gas had been used instead.

Reviewer: ```suggestion
  the initiating party, or a default value of 20 000 gas would be used instead.
```

* State Channel FSM: adds support for an optional `gas_price` parameter for
  all on-chain transactions, including the channel create one. This
  `gas_price` is to be used in transaction fee computation: the fee is

>   `gas_price * transaction_gas_cost`. The `transaction_gas_cost` is a subject

Reviewer: ```suggestion
  `gas_price * transaction_gas_cost`. The `transaction_gas_cost` is subject
```
